contract-addresses/permissions.json

```diff
@@ -28,8 +25,7 @@
     "aaveEthPCVDripController",
     "compoundEthPCVDripController",
     "pcvGuardian",
-    "daiPCVDripController",
```

Comment:
If the `daiPCVDripController` cannot pull DAI from the CompoundDAIPCVDeposit, then this whole system will not work.

We need to grant the PCVController role to the daiPCVDripController.